tests/python_functional/src/syslog_ng_config/statements/destinations/snmp_destination.py

```diff
@@ -21,9 +21,17 @@
 #
 #############################################################################
 from src.syslog_ng_config.statements.destinations.destination_driver import DestinationDriver
+from src.syslog_ng_ctl.driver_stats_handler import DriverStatsHandler
 
 
 class SnmpDestination(DestinationDriver):
     def __init__(self, **options):
         self.driver_name = "snmp"
         super(SnmpDestination, self).__init__(None, options)
+        self.stats_handler = DriverStatsHandler(group_type=self.group_type, driver_name=self.driver_name)
```

Comment:
`DriverStatsHandler` class should be initialized in this way in `DestinationDriver` class, because other dest drivers 
can also use this `DriverStatsHandler` object.

Reply:
done